doc/docs/additional.md

```diff
@@ -1,91 +1,73 @@
 # Additional topics
 
 ## Authentication
+
 Any openIDConnect provider can be used to supply oAuth credentials.
 
 The required set of parameters are:
 
-* oidConnectUrl
-* clientId
-* clientSecret
-* oAuthUsername
-* oAuthPassword
+- oidConnectUrl
+- clientId
+- clientSecret
+- oAuthUsername
+- oAuthPassword
 
 You can add oAuth authentication to the pipeline by providing the required set
-of parameters via the command line or in
-the `/pipelines/controller/config/application.yaml` file.
+of parameters via the command line or in the
+`/pipelines/controller/config/application.yaml` file.
 
 === "Cmd line"
-    ```java
-    $ java -cp ./pipelines/batch/target/batch-bundled-0.1.0-SNAPSHOT.jar org.openmrs.analytics.FhirEtl \
-        --fhirServerUrl=[FHIR_SERVER_URL] --outputParquetPath=[PATH] \
-        --resourceList=Patient,Encounter,Observation --batchSize=200 \
-        --clientId=[CLIENT_ID] --clientSecret=[CLIENT_SECRET] \
-        --OAuthUsername=[USERNAME] --OAuthPassword=[PASSSWORD] \
-        --oidConnectUrl= [OPENID_CONNECT_URL] \
-    ```
+`java $ java -cp ./pipelines/batch/target/batch-bundled-0.1.0-SNAPSHOT.jar org.openmrs.analytics.FhirEtl \ --fhirServerUrl=[FHIR_SERVER_URL] --outputParquetPath=[PATH] \ --resourceList=Patient,Encounter,Observation --batchSize=200 \ --clientId=[CLIENT_ID] --clientSecret=[CLIENT_SECRET] \ --OAuthUsername=[USERNAME] --OAuthPassword=[PASSSWORD] \ --oidConnectUrl= [OPENID_CONNECT_URL] \ `
```

Comment:
Is it possible to configure spotless not to make changes in the code parts? I think the old version is more readable.